versioned_docs/version-5.x/navigation-actions.md

```diff
@@ -44,6 +44,37 @@ navigation.dispatch(
 );
 ```
 
+The `navigate` action may also take on a different function signature:
+
+`CommonActions.navigate(name, navigateParams)`
+
+- `name` - _string_ - A destination name of the route that has been defined somewhere
+- `navigateParams` - _object_ - An object which may contain any of the following fields:
+  - `screen` - _string_ - The screen of a nested navigator to route to
+  - `params` - _object_ - Params to merge into the destination route
+  - `initial` - _boolean_ - <!-- HELP WANTED: Still didn't figure out what this property does - I see it's used in packages/core/src/useNavigationBuilder.tsx but I didn't yet understand what it's purpose is. Could someone explain? I'll then replace this comment. -->
```

Comment:
What does the `initial` property do? I'll change this line if you reply with the usage of it. Thanks.